wolt_services.py

```diff
@@ -0,0 +1,21 @@
+import requests
+from constants import WOLT_URL
+
+
+def check_if_restaurant_is_open(
+    restaurant_id: str, lat: float, lon: float
+) -> bool | str:
+    wolt_url = (
+        WOLT_URL.replace("<RESTAURANT_ID>", restaurant_id)
+        .replace("<LATITUDE>", str(lat))
+        .replace("<LONGITUDE>", str(lon))
+    )
+    try:
+        response = requests.get(wolt_url)
+        response.raise_for_status()  # Raise an exception for bad status codes
+
+        data = response.json()
+        return data["venue"]["open_status"]["is_open"]
+
+    except requests.exceptions.RequestException:
+        return "could not get data from wolt"
```

Comment:
As said, either handle the error here (if possible - depends on the nature of the issue. Retry? ignore? you tell me), treat it as a "no-open" if makes sense or handle outside. Never return an "indicative" message string like this.